pkg/proxy/ipvs/proxier.go

```diff
@@ -296,6 +297,10 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
 	return ips, nil
 }
 
+func (r *realIPGetter) BoundIPs() (sets.String, error) {
+	return r.nl.GetLocalAddresses(DefaultDummyDevice, "")
```

Comment:
I think `ListBindAddress` would achieve the same thing in this case since we pass in an empty filter device? Was there a reason for using GetLocalAddress or was it arbitrary?

Reply:
I figured using a set felt more natural for this test later: `boundAddresses.Has(vs.Address.String())`